kopflos: resource loaders (#32)
zazuko/data-centric.zazuko.com: docs/apis/kopflos/how-to/resource-loaders.md

# Configure resource loaders

As explained on the [Request pipeline page](../explanations/request-pipeline.md), early in the request pipeline, Kopflos selects a resource loader to load the requested resource's Core Representation.

Resource Loaders can be declared on instances of `kopflos:ResourceShape`, `kopflos:Api`, and `kopflos:Config` resources.

## Predefined loaders

Kopflos recognizes two loaders out of the box. They are identified by their shorthand identifiers.

### Own graph loader

The "Own Graph" loader is the default.

> It loads the requested resource from the same graph where the request was made.

Reviewer: the sentence is not clear to me
```suggestion
It loads the named graph for the requested resource.
```
